x-pack/plugins/ml/public/datavisualizer/datavisualizer.html

```diff
@@ -112,28 +156,49 @@ <h2 class="kuiSubTitle kuiVerticalRhythm">Metrics</h2>
 
       <div class="kuiPanel kuiVerticalRhythm datavisualizer-panel card-panel">
         <div class="euiText">
-          <h2 class="kuiSubTitle kuiVerticalRhythm">Fields</h2>
+          <h2
+            class="kuiSubTitle kuiVerticalRhythm"
+            i18n-id="xpack.ml.datavisualizer.fieldsTitle"
+            i18n-default-message="Fields"
+          ></h2>
         </div>
 
         <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--responsive euiFlexGroup--wrap">
           <div class="field-sort-controls euiFlexItem euiFlexItem--flexGrowZero">
             <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--responsive">
               <div ng-if="showAllFields === false" class="field-count-label euiFlexItem euiFlexItem--flexGrowZero">
-                <span class="field-emphasis">{{ fieldCards.length }}</span>
-                <ng-pluralize count="fieldCards.length" when="{'1':'field exists', 'other':'fields exist'}">fields exist</ng-pluralize> in documents
-                (<span class="field-emphasis">{{ totalNonMetricFieldCount }}</span> in total)
+                <span
+                  i18n-id="xpack.ml.datavisualizer.showMFieldDescription"  
+                  i18n-default-message="{wrappedFieldCardsCount} {fieldCardsCount, plural, one {field exists} other {fields exist}} in documents ({totalNonMetricFieldCount} in total)"
+                  i18n-values="{
+                      wrappedFieldCardsCount: '<span class=\'field-emphasis\'>' + fieldCards.length + '</span>',
+                      fieldCardsCount: fieldCards.length,
+                      totalNonMetricFieldCount: '<span class=\'field-emphasis\'>' + totalNonMetricFieldCount + '</span>'
+                  }"
+                ></span>
               </div>
 
+
               <div ng-if="showAllFields === true" class="field-count-label euiFlexItem euiFlexItem--flexGrowZero">
-                <span class="field-emphasis">{{ fieldCards.length }}</span>
-                <ng-pluralize count="fieldCards.length" when="{'1':'field', 'other':'fields'}">fields</ng-pluralize>
-                (<span class="field-emphasis">{{ populatedNonMetricFieldCount }}</span>
-                <ng-pluralize count="populatedNonMetricFieldCount" when="{'1':'exists', 'other':'exist'}">exist</ng-pluralize> in documents)
+                <span
+                  i18n-id="xpack.ml.datavisualizer.showAllFieldsDescription"
+                  i18n-default-message="{wrappedFieldCardsCount} {fieldCardsCount, plural, one {field} other {fields}} ({wrappedPopulatedNonMetricFieldCount} {populatedNonMetricFieldCount, plural, one {exists} other {exist}} in documents)"
+                  i18n-values="{
+                    wrappedFieldCardsCount: '<span class=\'field-emphasis\'>' + fieldCards.length + '</span>',
```

Comment:
Again `<span>` should not be inside here

Reply:
With the last **master** <code>\<span\></code>  tag render correctly
![image](https://user-images.githubusercontent.com/25555662/50217988-78063e00-039b-11e9-9d5b-434e4d6aa4aa.png)